migrations/versions/d415d490eb05_.py

```diff
@@ -0,0 +1,33 @@
+"""Add auth_count column to authcache table
```

Comment:
Can you please add in the first line at the beginning:

    v3.6: Add...

Since only the first line is displayed when running

    pi-manage db history

Reply:
Will do

Reply:
Done in 53bf2249